### PR TITLE
Quick fixes in authors' bio (third person). That should fix #353 

### LINKED ISSUE #353: Les bio doivent être à la 3e pers
J'ai vu celle de @awillemant à la 1e pers, je ferais un tour et ajusterai ça à l'occassion.
Faudra aussi documenter ça que les gens le fassent d'eux même.

### PR DESCRIPTION
**Pull request by Macxim**
Documentation is still to be written.

### CODE REVIEW

File: content/authors/magsout.json [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
       ]
     }
   },
   "en": {
     "bio": {
-      "short": "Front-end web developer",
+      "short": "Front-end web developer.",
       "long": [
-        "Front-end web developer @splitfire. Loves to hunt bugs at night, series-tv addict."
+        "Front-end web developer @splitfire. Loves to hunt bugs at night, TV-series addict."
```
Comment (MoOx) [inline content/authors/magsout.json]: TV show maybe :)

**Comment by MoOx** [comment]
Nice !

File: content/authors/magsout.json (hunk 1 above)
Comment (Macxim) [inline content/authors/magsout.json]: You're right. That sounded weird indeed.

**Comment by MoOx** [comment]
Great !

**Pull request merged**